balrogscript/script.py

```diff
@@ -95,7 +95,7 @@ def submit_locale(task, config, auth0_secrets):
             # Get release metadata from manifest
             submitter, release = create_locale_submitter(e, suffix, auth0_secrets, config)
             # Connect to balrog and submit the metadata
-            retry(lambda: submitter.run(**release), jitter=2, sleeptime=2, max_sleeptime=5, attempts=10)
+            retry(lambda: submitter.run(**release), jitter=5, sleeptime=10, max_sleeptime=30, attempts=10)
```

Comment:
I'm not sure sleeptime is high enough, but let's try it. Getting more failures wouldn't be the worst thing, as I should be able to track them down better with https://github.com/mozilla-releng/balrogscript/pull/71 landed.

Reply:
Yeah, initially I was thinking at `20`. Let's see how this goes and then we can re-adjust 👍